astropy/modeling/tests/test_models.py

```diff
@@ -23,14 +23,18 @@
 from ..core import FittableModel
 from ..polynomial import PolynomialBase
 from ...tests.helper import pytest
-
+from ...utils import minversion
 
 try:
+    import scipy
     from scipy import optimize  # pylint: disable=W0611
     HAS_SCIPY = True
 except ImportError:
     HAS_SCIPY = False
 
+if HAS_SCIPY and not minversion(scipy, "0.14"):
+    HAS_SCIPY = False
```

Comment:
Or maybe just somehow use `tabular.has_scipy` specifically for `Tabular` tests without modifying `HAS_SCIPY` here.